src/main/java/lotto/auto/domain/Lotto.java

```diff
@@ -2,20 +2,17 @@
 
 import lotto.auto.vo.WinNumber;
 
-import java.util.Collections;
 import java.util.List;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
 
 public class Lotto {
-
-    //여기에 static 변수를 넣으면 value update 가 되지 않아 로또를 생성할 때 같은 번호가 리스트에 들어갑니다 ㅠㅠ static 을 쓰면서 어떻게 구현해야 할 지 모르겠습니다.
-    private final List<Integer> entireNumber = IntStream.rangeClosed(1, 45).boxed().collect(Collectors.toList());
     private final List<Integer> lottoNumber;
 
     public Lotto() {
-        Collections.shuffle(entireNumber);
-        this.lottoNumber = entireNumber.subList(0, 6);
+        this.lottoNumber = new FullNumber().getFullNumber().subList(0, 6);
```

Comment:
```suggestion
        this(new FullNumber().getFullNumber().subList(0, 6));
```
부생성자는 주생성자를 호출하도록 구현하는 습관을 들이면 어떨까?